src/main/java/org/cascadebot/cascadebot/commands/core/InviteCommand.java

```diff
@@ -17,8 +17,10 @@
     public void onCommand(Member sender, CommandContext context) {
         if (Environment.isProduction()) {
             context.getDirectMessageMessaging().replyDM(CascadeBot.getInvite(), true);
+            context.getTypedMessaging().replyInfo("We've sent you a DM with more information!");
         } else {
             context.getDirectMessageMessaging().replyDM("https://www.youtube.com/watch?v=ARJ8cAGm6JE");
+            context.getTypedMessaging().replyInfo("We've sent you a DM with more information!");
```

Comment:
Shouldn't this be localized?

Also could just put this line after the if else.